src/wallet.js

```diff
@@ -1122,7 +1122,6 @@ const wallet = {
    */
   resetWalletData() {
     this.cleanWallet();
-    this.cleanServer();
```

Comment:
I feel we should add a test to confirm the server is not being erased.

Reply:
Done!